package/MDAnalysis/coordinates/chain.py

```diff
@@ -56,13 +56,99 @@
 from . import core
 
 
+def multi_level_argsort(l):
+    """
+    This sorts based on the start time.
+
+    Parameters
+    ----------
+    l : list
+
+    Returns
+    -------
+    indices
+    """
+    return [el[0] for el in sorted(enumerate(l), key=lambda x: x[1][0])]
+
+
+def filter_times(times, dt):
+    """Given a list of start and end times this function filters out any duplicate
+    time steps preferring the last tuple.
+
+    ((0, 3), (0, 3)) -> (1, )
```

Comment:
Format as a pre-block or Python code.

Reply:
Also, the example implies that tuples are returned but the code returns lists. Please make docs and code agree.